.github/workflows/nightly-tests.yml

```
@@ -1,51 +1,27 @@
-name: Rust
-
+name: Nightly tests
 on:
-  push:
-  pull_request:
-    branches: [ "main" ]
+  schedule:
```

```suggestion
  workflow_dispatch:
  schedule:
```
I think this might enable us to also run it manually

---

done